TangoTest.cpp

```diff
@@ -2133,14 +2133,17 @@ void TangoTest::write_string_spectrum(Tango::WAttribute &attr)
   attr.get_write_value(p);
 
   int len = attr.get_w_dim_x();
-  if (len > kSpecLen)
-    len = kSpecLen;
+  len = (len <= kSpecLen) ? len : kSpecLen;
```

Comment:
I'm personally not convinced by the need of this change (line 2136) because at the end, it does the same thing as before, except that when len <= kSpecLen, it does
len = len; 
which is not needed obviously. This will **strongly** affect the performances!!! I cannot accept this change of course... ;-)